.github/workflows/_finalize.yaml

```diff
@@ -8,7 +8,11 @@ on:
         description: Update the landing page badges with run outcomes
         default: false
         required: false
-
+      ARTIFACT_NAME:
+        type: string
+        description: 'Name of the artifact zip file'
+        required: false
+        default: 'artifact-final-job-status'
```

Comment:
Can we make the default name more intuitive so that it standup out among a bunch of artifacts that "this is THE final report"?

Reply:
something like: artifact-the-final-report?